packages/pluggableWidgets/image-viewer-web/tests/e2e/specs/onClick.spec.ts

```diff
@@ -0,0 +1,56 @@
+import page from "../../../../../../configs/e2e/src/pages/page";
+import ImageViewer from "../objects/imageViewer.widget";
+import * as html from "../objects/constants";
+
+// eslint-disable-next-line jest/no-disabled-tests
+xdescribe("Image viewer", () => {
```

Comment:
Did you include notes/ stories to enable e2e tests for image viewer?

Reply:
Not yet! I'll create a new task to enable/fix the events and the design properties tests.